app/views/blog/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= @Blog %>
```

Comment:
What is this currently displaying?  Keep in mind you are calling on the instance variable in your index controller method, which will be an Active Record query of an array

Reply:
Fixed!